eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -109,6 +109,17 @@ jobs:
               -DBUILD_TRANSPORT_CURL=OFF
               -DBUILD_DOCUMENTATION=YES
 
+        - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
```

Comment:
May want to move this lower in the file where the other publishing happens.

Reply:
Yeah, my comment speaks to why I did it this way.